changes/change_GLT-4423.md

```diff
@@ -0,0 +1,5 @@
+The following fields will now accept numbers with up to 6 digits before the decimal.
+* "Conc." field in Samples, Libraries, Library Aliquots, Pool.
```

Comment:
This is displayed as "Conc." on the bulk pages, but "Concentration" on the form pages, so it's probably clearer to call it "Concentration" here